docs/CMakeLists.txt

```diff
@@ -11,14 +11,12 @@ if("${DOXYGEN_VERSION}" VERSION_GREATER_EQUAL 1.9.4 AND CMAKE_CXX_COMPILER_ID ST
 # Doxygen <= 1.9.4 suffers from https://github.com/doxygen/doxygen/issues/8784
 set(DOXYGEN_CLANG_ASSISTED_PARSING YES)
 set(DOXYGEN_CLANG_DATABASE_PATH "${CMAKE_BINARY_DIR}")
-else()
-set(DOXYGEN_WARN_AS_ERROR YES)
 endif()
 set(DOXYGEN_DISABLE_INDEX YES)
 set(DOXYGEN_ENUM_VALUES_PER_LINE 1)
 set(DOXYGEN_EXAMPLE_PATH "${DDC_SOURCE_DIR}/examples")
 set(DOXYGEN_EXPAND_ONLY_PREDEF YES)
-set(DOXYGEN_EXTRACT_ALL NO)
+set(DOXYGEN_EXTRACT_ALL YES)
```

Comment:
This cannot be changed. See the doxygen documentation cited in the description of #363